certora/conf/CER-1-Owner/CER-60-Owner-addressPrefix.conf

```diff
@@ -0,0 +1,19 @@
+{
+    "files": [
+        "certora/harness/EthereumVaultConnectorHarness.sol"
+    ],
+    "verify": "EthereumVaultConnectorHarness:certora/specs/CER-1-Owner/CER-60-Owner-addressPrefix.spec",
+    "packages": [
+        "@openzeppelin=lib/openzeppelin-contracts"
+    ],
+    "prover_args": [
+        "-useBitVectorTheory"
+    ],
+    "solc_via_ir" : true,
+    "assert_autofinder_success" : true,
+    "optimistic_loop": true,
+    "optimistic_hashing": true,
+    "solc_optimize" :  "10000",
+    "msg": "CER-1-Owner/CER-60-Owner-addressPrefix",
+    "solc": "solc8.20"
+}
```

Comment:
please run with --rule_sanity basic in all conf files

Reply:
fixed